akka-docs-dev/rst/java/stream-quickstart.rst

```diff
@@ -56,13 +63,17 @@ For now let's simply print each author:
 

```

Comment:
The paragraph above is lacking materialized value type parameter on Sink and Source types. We should have add it at least for consistency.